### ISSUE TITLE
Database' object has no attribute '_pid

### ISSUE THREAD

**Issue by santoshs** [robinedwards/neomodel#226]
When running as a standalone python application, everything works fine. Once the application is hosted as a WSGI on apache2, I get the above error. The backtrace is pasted below.

```
Traceback (most recent call last):
  <..snip>
  File "/var/www/contacts/models/models.py", line 10, in <module>
    class Contact(StructuredNode):
  File "/usr/lib/python2.7/site-packages/neomodel/core.py", line 58, in __new__
    install_labels(inst)
  File "/usr/lib/python2.7/site-packages/neomodel/core.py", line 20, in install_labels
    cls.__label__, key))
  File "/usr/lib/python2.7/site-packages/neomodel/util.py", line 28, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/lib/python2.7/site-packages/neomodel/util.py", line 86, in cypher_query
    if self._pid != os.getpid():
AttributeError: 'Database' object has no attribute '_pid'
```

This application uses flask, and neomodel version 3.0.1.

**Comment by robinedwards** [comment]
Hey thanks for getting in touch. I've fixed this on master already. Can you try it and see if it works?

I will be releasing to pypi after Christmas

**Comment by robinedwards** [comment]
Just released to pypi 3.0.2 fixes the issue
